paper/paper.md

```diff
@@ -63,11 +63,11 @@ Platforms such as MassBank [@horai_massbank_2010], GNPS [@Wang2016] or MetaboLig
 Without suitable tools, however, exploitation of such datasets remains overly challenging. 
 In particular, large collected datasets contain data acquired using different instruments and measurement conditions, and can further contain a significant fraction of inconsistent, wrongly labeled, or incorrect metadata (annotations).
 
-``matchms`` is an open-access Python package to import, process, clean, and compare mass spectrometry data (MS/MS) (see \autoref{fig:flowchart}).
+``matchms`` is an open-source Python package to import, process, clean, and compare mass spectrometry data (MS/MS) (see \autoref{fig:flowchart}).
```

Comment:
I think you mean open source here (not open access).

Reply:
Yes, indeed. Thanks!